### ISSUE TITLE
When opening the asciidoc in a browser via the embedded server, interactive SVG images are not found

### ISSUE THREAD

**Issue by bric3** [asciidoctor/asciidoctor-intellij-plugin#360]
The asciidoc plugin offers the ability to open the rendered document in a browser, which is working quite fine. But in some cases there's some 404s, e.g. when working with SVG images.

## The following snippets are fine

```adoc
.30s profile `itimer` event
image::app-2511384363-8tpw5-itimer.svg[itimer,300,opts="inline"]
```

In this case the SVG is within the document.

```adoc
.30s profile `itimer` event
image::app-2511384363-8tpw5-itimer.svg[itimer,300]
```

In this case the image URL is `http://localhost:63342/ead61b63-b0a6-4ff2-a49a-86be75ccfd1a/image?file=%2FUsers%2Fbric3%2Fgit-repo%2Fsrc%2Fdocs%2Fasciidoc%2Fdev%2Fapp-2511384363-8tpw5-alloc.svg&mac=DrYm01qtCSBmxLkcprIyL5jrDvOAU6FuXypDawW9zNU=&hash=4175a2f55309ecf14ff106efc6622149`

![Screen Shot 2019-10-29 at 12 24 41](https://user-images.githubusercontent.com/803621/67763500-1e08e700-fa48-11e9-82ee-60b6274a1306.png)

## What does not work :

But when the SVG has the `interactive` option, then the SVG is not found

```adoc
.30s profile `itimer` event
image::app-2511384363-8tpw5-itimer.svg[itimer,300,opts="inline"]
```

![Screen Shot 2019-10-29 at 12 25 46](https://user-images.githubusercontent.com/803621/67763210-712e6a00-fa47-11e9-979d-882b401948e0.png)

And in this case too, the linked image is not found

```adoc
.30s profile `itimer` event
[link=app-2511384363-8tpw5-itimer.svg]
image::app-2511384363-8tpw5-itimer-small.png[itimer,role="right"]
```

This generated link : `http://localhost:63342/ead61b63-b0a6-4ff2-a49a-86be75ccfd1a/app-2511384363-8tpw5-itimer.svg`

----

AsciiDoc Plugin Version: 0.30.14
IDE Name and Version: 2019.3 EAP

**Comment by ahus1** [comment]
I'll have a look how to make link=... work. I already have an idea how to do that. 

I am not sure what the SVG `interactive` option is, I'm probably not very into SVGs. Can you please explain and provide an example file that is not working (I assume this is something within the SVG)?

**Comment by ahus1** [comment]
@bric3 - regarding your interactive not working example, should this be "interactive" in the opts, not "inline"?

```adoc
.30s profile `itimer` event
image::app-2511384363-8tpw5-itimer.svg[itimer,300,opts="interactive"]
```

**Comment by bric3** [comment]
@ahus1 I followed the guide here: https://asciidoctor.org/docs/user-manual/#taming-svgs

**Comment by ahus1** [comment]
Thanks, I've used these example to test it. In the next version the example will work both in the JavaFX and the browser preview. 

The JavaFX preview has its limitations regarding SVG, some graphics look quite crippled. Unfortunately this is nothing I can change. If your SVG looks crippled in the JavaFX preview inside IntelliJ, please switch to the browser.

**Comment by ahus1** [comment]
I've published pre-release 0.30.15. Please comment if this works for you.
The new pre-release of the plugin is available from [GitHub releases](https://github.com/asciidoctor/asciidoctor-intellij-plugin/releases) and the [IntelliJ AsciiDoc EAP repository](https://github.com/asciidoctor/asciidoctor-intellij-plugin/blob/master/CONTRIBUTING.adoc#eap).

**Comment by bric3** [comment]
Hi thanks for the quick fix!
Yes that's why I was using the external browser, since this SVG is rather complex.
